lib/node_modules/@stdlib/stats/base/smin/README.md

```diff
@@ -160,6 +145,123 @@ console.log( v );
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/smin.h"
+```
+
+#### stdlib_strided_smin( N, \*X, strideX )
+
+Calculate the minimum value of a single-precision floating-point strided array.
```

Comment:
```suggestion
Computes the minimum value of a single-precision floating-point strided array.
```

Function descriptions should always use the "indicative mood". For the top-level README description, package.json, and `index.js`, use the imperative mood.